sklearn/cluster/k_means_.py

```diff
@@ -313,12 +320,25 @@ def k_means(X, n_clusters, init='k-means++', precompute_distances='auto',
     x_squared_norms = row_norms(X, squared=True)
 
     best_labels, best_inertia, best_centers = None, None, None
+    if n_clusters == 1:
```

Comment:
Does it make sense to return the mean, inertia directly for this case?

Because the default initialization "kmeans++" is the same as picking up a random center in this case.